docs/extras/modules/data_connection/text_embedding/spacy_embedding.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the necessary classes\n",
```

Comment:
notebooks should have a title of `# Spacy` and then all other cells should NOT have `#` but should be more like `## Import the ....`

Reply:
> notebooks should have a title of `# Spacy` and then all other cells should NOT have `#` but should be more like `## Import the ....`

done. Notebook updated